Publisher/nl/database-export.md

```diff
@@ -1,66 +1,39 @@
 # Exporteren van profielgegevens
-Naast importeren is het ook mogelijk om profielen en subprofielen te exporteren.
-Je kunt een hele database in één keer exporteren, of in onderdelen
-(bijvoorbeeld per collectie). Bij een export wordt een bestand in een formaat
-naar keuze gegenereert, dat je kunt downloaden of laten e-mailen.
-
-Als je graag in real-time de clicks, opens e.d. van profielen wilt ontvangen,
-kun je ook een [webhook](./webhooks) instellen.
+Naast het importeren van gegevens is het mogelijk om (sub)profielgegevens te exporteren. Daarbij kun je de gehele database in één keer of in onderdelen exporteren (bijvoorbeeld per selectie, collectie of miniselectie). Wanneer een exportbestand is aangemaakt kun je deze downloaden of laten versturen naar een e-mailadres.
 
 ## Exportbestand voorbereiden
-Klik op de database of collectie waarvan je een export wilt maken en klik vervolgens op **Exports** in de toolbar.
-Geef je export een naam en kies de velden die je wilt exporteren. In de vervolgstappen kun je nog specifieke instellingen meegeven aan de export. Hierbij kun je denken aan het bestandsformaat, het scheidingsteken en de codering. Als er in je database profielen voorkomen die zelf een scheidingsteken in hun velden hebben staan (bijvoorbeeld een harde tab in een woonplaatsveld), kies dan bij *kolominstelling* voor de optie *"Kolommen met aanhalingstekens"*. De waardes worden dan
-tussen quotes geplaatst zodat er geen conflicten met het scheidingsteken optreden.
+Je kunt een export starten door in de menubalk van een database, selectie, collectie of miniselectie te kiezen voor '**Exports**'. De exportconfiguratie bevat een aantal stappen die je moet doorlopen voordat de export gestart kan worden. Dat zijn:
+
+**Info**  
+Hier geef je de export een naam en een eventuele beschrijving. Dat helpt je om periodieke exports beter uit elkaar te houden.
+
+**Velden**  
+Hier selecteer je de velden die je vanuit de Copernica-omgeving wilt exporteren.
+
+**Filter**  
+Hier geef je aan of je alle (sub)profielen wilt exporteren of enkel degene die zijn aangemaakt of bewerkt sinds de vorige export.
+
+**Bestand**  
+Hier geef je het bestandsformaat (XLS, TXT, CSV of XML) en de encoding (standaard UTF-8) van het bestand aan. Daarnaast bepaal je of het bestand ingepakt moet worden als ZIP-bestand. Dit raden wij aan bij het exporteren van een grote hoeveelheid gegevens om het bestand zo klein mogelijk te houden.
```

Comment:
Vier keer achter elkaar begint de zin met 'hier', iets meer variatie is wenselijk